### PR TITLE
Allow configuring deviceId in config

### PR DESCRIPTION
**Pull request by passcod**
### Changes

Make it possible to specify a deviceId in the facility server config. This is used by the auto-deploys.

### Checklist

- [x] Code is finished

_Upon merging:_

- [ ] Update the changelog (find it [here](https://github.com/beyondessential/tamanu/releases) in the appropriate _draft_ release)
  - [ ] with a ticket reference (e.g. `TAN-123: a one line description`, keep the lists sorted)
  - [ ] any config/settings changes and manual deployment steps
- [ ] Update the [config reference](https://beyond-essential.slab.com/posts/reference-config-file-0c70ukly) as needed

### CODE REVIEW

**Review by edmofro — APPROVED** [review]
I realise this is still in draft, but fwiw lgtm. Approving so it no longer requires codeowner

**Review by NavarroEmilioLuis — APPROVED** [review]
LGTM. Though this means we can never change it once we start... Though I guess we'll do that through the database + config?

**Pull request merged**
